Documentation/policy/language.rst

```diff
@@ -39,15 +39,6 @@ cluster managed by Cilium. Label-based L3 policies are defined by using an
 received (on ingress), or sent (on egress). An empty `EndpointSelector` allows
 all traffic. The examples below demonstrate this in further detail.
```

Comment:
`An empty `EndpointSelector` allows all traffic.` might need to hint that it's namespace specific since we only mention that in the namespaces section.

Reply:
Good point, will clarify.